src/sentry/static/sentry/app/api.jsx

```diff
@@ -59,13 +58,13 @@ export class Client {
    * If so, redirect user to new project slug
    */
   hasProjectBeenRenamed(response) {
-    let code = response && idx(response, _ => _.responseJSON.detail.code);
+    let code = response?.responseJSON?.detail?.code;
```

Comment:
😍 Do we start using es features when they hit stage 2? [optional chaining operators](https://github.com/TC39/proposal-optional-chaining) hasn't gone past stage 2 yet.

Reply:
Mostly following @billyvg's lead here https://github.com/getsentry/sentry/pull/9666